components.json

```diff
@@ -6,12 +6,12 @@
   "tailwind": {
     "config": "tailwind.config.ts",
     "css": "src/styles/globals.css",
-    "baseColor": "slate",
-    "cssVariables": false,
+    "baseColor": "gray",
+    "cssVariables": true,
```

Comment:
why you change this to gray

Reply:
ตอน install Table กดผิดเลือกทุกอย่างเป็น  gray มั้งนึกว่ามันเป็นแค่ของ Table

Reply:
> ตอน install Table กดผิดเลือกทุกอย่างเป็น gray มั้งนึกว่ามันเป็นแค่ของ Table



> ตอน install Table กดผิดเลือกทุกอย่างเป็น gray มั้งนึกว่ามันเป็นแค่ของ Table

แก้ลพ commit ใหม่ส่วนอื่นกำลังเช็ค

Reply:
มันอยู่ไฟล์ไหนนะ